packages/block-editor/src/components/rich-text/test/index.native.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+import { RichText } from '../index.native';
```

Comment:
I imported from `../index.native` because I was originally using this within the `gutenberg` test pipeline (without altering the jest config). I'm unsure whether this will be necessary if we plan to introduce a separate mobile-specific jest configuration which resolves all `*.native.js` files to their `*.js` key-names in the dependency map. WDYT?

Reply:
Running with `gutenberg-mobile` tooling, importing `../index` will be enough 👍

Reply:
Thanks @etoledom ! Fixed :heavy_check_mark: